deep_speech_2/error_rate.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+"""This module provides functions to calculate error rate in different level.
+e.g. wer for word-level, cer for char-level.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import numpy as np
+
+
+def levenshtein_distance(ref, hyp):
```

Comment:
1. rename it to `_levenshtein_distance` ?
2. Add a simple description or reference for levenshtein_distance?